src/position/mod.rs

```diff
@@ -203,7 +203,11 @@ pub trait Positionable: Sized {
     }
 
     /// Set the position relative to the widget with the given widget::Index.
-    fn x_y_relative_to<I: Into<widget::Index> + Copy>(self, other: I, x: Scalar, y: Scalar) -> Self {
+    fn x_y_relative_to<I: Into<widget::Index> + Copy>(self,
```

Comment:
I'd probably prefer to move the `Into<widget::Index>` bound into a where clause (like the following methods) over putting all these args on their own lines.